lib/Steps/Integrations/NextJs.ts

```diff
@@ -68,6 +72,30 @@ export class NextJs extends BaseIntegration {
       nl();
     }
 
+    const selectedProjectSlug: string | null = answers.config?.project?.slug;
+    if (selectedProjectSlug) {
+      const hasFirstEvent = answers.wizard?.projects?.find?.((p: { slug: string; }) => p.slug === selectedProjectSlug)?.firstEvent;
+      if (!hasFirstEvent) {
```

Comment:
I don't remember what the answers look like, but the only reason you get the project slug is to check if there's already an event. Assuming the project is selected after fetching them from Sentry, can you get that `firstEvent` directly? Something like:
```js
const hasFirstEvent = answers.config?.project?.firstEvent;
```

Reply:
Heads up if you also want to target big orgs: there's a limit of max 50 projects fetched per org in the API, [see](https://github.com/getsentry/sentry/blob/cdfe392ddbfa78c1560f51ce85f0d39c3b42ab00/src/sentry/web/frontend/setup_wizard.py#L47). Projects are sorted by date added desc, so it shouldn't be a problem though. Feel free to land this as-is, and open a follow-up PR if you want to add further changes.

Reply:
When I tested this, answers.config.project only has the slug property and nothing else. That's why I had to do this lookup.